### PR TITLE
Transfer ownership

### PR DESCRIPTION
**Pull request by florisdobber**
Ticket: [https://trello.com/c/f1GMCSeX](https://trello.com/c/f1GMCSeX)
transfer ownership
confirmation

### CODE REVIEW

File: src/api/api.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ export const createTeam = async team => AxiosInstance.post(TEAM, team);
 export const joinTeam = async id => AxiosInstance.post(`${TEAM}/${id}/join`);
 export const leaveTeam = async id => AxiosInstance.post(`${TEAM}/${id}/leave`);
 export const kickMember = async (team, member) => AxiosInstance.post(`${TEAM}/${team}/members/${member}/kick`);
+export const transferOwnership = async (team, member) => AxiosInstance.post(`${TEAM}/${team}/transfer_ownership`, member);
```
Comment (jackblanc) [inline src/api/api.js:8]: Can you change "team" to "id" to match the other calls?

File: src/views/IndividualTeamView.vue [hunk 2]
```diff
@@ -186,8 +181,40 @@ export default {
...
+      }).then((team) => {
+        this.team = team.data;
+        this.loaded = true;
+      }).catch((error) => {
+        // eslint-disable-next-line
+        console.log(error.message);
+      });
+    },
+    confirmTransfer(member) {
+      this.$bvModal.msgBoxConfirm(`Are you sure you would like to make ${member.username} team leader?`, {
```
Comment (jackblanc) [inline src/views/IndividualTeamView.vue:193]: This is great 👍

File: src/views/IndividualTeamView.vue [hunk 3]
```diff
@@ -186,8 +181,40 @@ export default {
...
+        footerClass: 'p-2 border-top-0',
+        centered: true,
+      }).then((confirmed) => {
+        if (confirmed) {
+          return this.makeLeader({ newLeaderId: member.id });
+        }
+        return '';
+      }).catch((error) => {
+        // eslint-disable-next-line
+        console.log(error.message);
```
Comment (jackblanc) [inline src/views/IndividualTeamView.vue:208]: At some point, we will have to replace these with errors the user sees, but it's probably best to do that all at once? 

cc @liammoyn

Comment (florisdobber) [reply src/views/IndividualTeamView.vue:208]: I can make a separate ticket for someone to go through and display error messages from every back end call. It'll probably be a lot of copy paste anyway.

Comment (liammoyn) [reply src/views/IndividualTeamView.vue:208]: Yep separate ticket is the way to go. Keep things consistent

**Pull request merged**